src/System.IO/src/System.IO.csproj

```diff
@@ -17,6 +17,13 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
```

Comment:
I missed the fact that this one is a partial facade. Yes you will need to stick with the project reference.